.gitignore

```diff
@@ -2,6 +2,8 @@
 logs
 *.log
 
+dist/*
```

Comment:
The build ends up with our Microsoft endpoints in the public distro - which is not really bad (the endpoints are public) - but kinda useless for the broader community and weird.

Reply:
`dist` is generated by running `npm run dist`

Ideally we want to avoid having generated artifacts in the repo unless there is a good reason to. 

These generated artifacts will include stuff such as our configuration, which is not really useful for third parties forking our repo. For example, `\dist\assets\app.js`, will contain the link to IcM, clientId, etc.